private_gpt/open_ai/openai_models.py

```diff
@@ -48,8 +52,11 @@ class OpenAICompletion(BaseModel):
     choices: list[OpenAIChoice]
 
     @classmethod
-    def from_text(
-        cls, text: str | None, finish_reason: str | None = None
+    def from_text_and_sources(
```

Comment:
tiny nit, but if you name the function `from_text_and_sources` I would expect the sources to be non mandatory, and `from_text` to still be there but with the sources optional

Reply:
Done!